node/src/BaseClient.ts

```diff
@@ -228,6 +228,44 @@ export type ReturnType =
     | ReturnTypeAttribute
     | ReturnType[];
 
+export type GlideString = string | Uint8Array;
+
+/**
+ * Enum representing the different types of decoders.
+ */
+export const enum Decoder {
+    /**
+     * Decodes the response into a buffer array.
+     */
+    Bytes,
+    /**
+     * Decodes the response into a string.
+     */
+    String,
+}
+
+/**
+ * Our purpose in creating PointerResponse type is to mark when response is of number/long pointer response type.
+ * Consequently, when the response is returned, we can check whether it is instanceof the PointerResponse type and pass it to the Rust core function with the proper parameters.
+ */
+class PointerResponse {
+    pointer: number | Long | null;
+    // As Javascript does not support 64-bit integers,
```

Comment:
Since JS does not support u64, why you're trying to store the pointer on line 252?